Cargo.lock

```diff
@@ -6214,10 +6213,9 @@ version = "0.3.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c2702e08a7a860f005826c6815dcac101b19b5eb330c27fe4a5928fec1d20ddd"
 dependencies = [
- "itoa 1.0.1",
+ "itoa 1.0.2",
  "libc",
  "num_threads",
- "quickcheck 1.0.3",
```

Comment:
This is clever, it looks like Cargo gained the ability to exclude `dev-dependencies` from the lockfile